__sdk__.js

```diff
@@ -6,5 +6,10 @@ module.exports = {
         entry:           './src/index',
         staticNamespace: '__muse__',
         automatic:        true
+    },
+    'tracker': {
```

Comment:
is there any naming convention to be followed here?

Reply:
I'd suggest we use `cart-tracker` here too, since we'd be expecting merchants to pass this explicitly in the sdk url without `automatic: true`.

Reply:
I think the tracker was gonna end up being used for more than just cart stuff, though. Correct me if I'm wrong @shivinkapur @bryanjenningz

Reply:
OK. If that's the case then makes sense. 👍